package.json

```diff
@@ -11,9 +11,8 @@
     "start": "strapi start",
     "build": "strapi build",
     "strapi": "strapi",
-    "strapi:test": "strapi develop --no-build",
     "test": "playwright test",
-    "test:postgres": "node ./init/test-postgres.js",
+    "test:postgres": "TEST_CONTAINER=true nyc playwright test",
```

Comment:
Instead of using script and spawn child process, we can start a container inside setup and stop it in teardown